src/main/java/agh/ics/oop/AbstractWorldMap.java

```diff
@@ -15,19 +15,24 @@ public AbstractWorldMap(){
     @Override
     public boolean place(Animal animal) {
```

Comment:
skoro rzucamy wyjątkiem to nie zwrócimy tu nigdy false -> ta metoda nie powinna już zwracać boolean